scripts/wiki/ddl.sql

```diff
@@ -0,0 +1,55 @@
+-- DDL for parsed Wiki data.
```

Comment:
It would be helpful to add a comment/description for each field.

Reply:
Added in next commit. Please close this conversation if you think the comments are sufficient.